<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileFrame</class>
 <widget class="QFrame" name="FileFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QComboBox" name="typeBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Net List</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Schematic</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fileBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxCount">
      <number>10</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fileBtn">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
